src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceControllerTest.kt

```diff
@@ -634,7 +634,7 @@ class LicenceControllerTest {
       licenceStatus = LicenceStatus.IN_PROGRESS,
       nomisId = "A1234AA",
       forename = "Bob",
-      surname = "Mortimer",
+      surname = "Bloggs",
```

Comment:
Don't want to ask an annoying question 😅  but are we def ok to use `Bloggs` and common names like `John` `Joe` `Jim` below? Is it safer to use `Some Person` or other def non names?

Reply:
I think our general take is that if a name has become an idiom for "an unknown person" (eg John Smith, Joe Bloggs, and John/Jane Doe) that it's probably fine, as it's clearly not a reference to an actual person.